### PR TITLE
Neuron SDK Release - January 30, 2021

### PR DESCRIPTION
**Pull request by aws-mesharma**
Pull request includes:

    Release notes updated for January 30, 2021 release of Neuron SDK.
    Fixes for the jupyter notebooks

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by micwade-aws — CHANGES_REQUESTED** [review]
Formatting issues in multiple compiler related rst files.  Mostly looks like md files written over the rst.

File: docs-rtd/release-notes/conda/torch/conda-torch-neuron.rst [hunk 1]
```diff
@@ -6,6 +6,40 @@ PyTorch-Neuron Conda Package Release notes
 This document lists the release notes for the Neuron Conda-Pytorch
 package.
 
+[1.7.1.1.3.0]
```
Comment (jeffhataws) [inline docs-rtd/release-notes/conda/torch/conda-torch-neuron.rst:9]: Should be [1.7.1.1.2.3.0]

File: docs-rtd/release-notes/conda/torch/conda-torch-neuron.rst [hunk 2]
```diff
@@ -6,6 +6,40 @@ PyTorch-Neuron Conda Package Release notes
...
+
+torch_neuron-1.7.1.1.2.3.0
+
+Resolved Issues
+---------------
+
+Resolved the segmentation fault when enabling profiling using NEURON_PROFILE=<directory> environment variable for inference within a PyTorch-Neuron Conda environment (https://github.com/aws/aws-neuron-sdk/issues/230). This fix will be available in the next DLAMI release.
+
+
+[1.5.1.1.2.1.0]
```
Comment (jeffhataws) [inline docs-rtd/release-notes/conda/torch/conda-torch-neuron.rst:26]: Should be [1.5.1.1.2.3.0]

File: docs-rtd/release-notes/conda/torch/conda-torch-neuron.rst [hunk 3]
```diff
@@ -6,6 +6,40 @@ PyTorch-Neuron Conda Package Release notes
...
+^^^^^^^^^^^^^^^
+Date: 1/30/2021
+
+Included Neuron Packages
+------------------------
+
+neuron-cc-1.2.1.0
+
+torch_neuron-1.5.1.1.2.1.0
```
Comment (jeffhataws) [inline docs-rtd/release-notes/conda/torch/conda-torch-neuron.rst:35]: Should be [1.5.1.1.2.3.0]

**Review by jeffhataws — CHANGES_REQUESTED** [review]

File: docs-rtd/release-notes/neuron-cc-ops/neuron-cc-ops-pytorch.rst [hunk 4]
```diff
@@ -12,6 +12,20 @@ python:
    print(*torch.neuron.get_supported_operations(), sep='\n')
 
 
+.. _pytorch-neuron-release-1220:
+
+PyTorch Neuron Release [1.2.2.0]
```
Comment (jeffhataws) [inline docs-rtd/release-notes/neuron-cc-ops/neuron-cc-ops-pytorch.rst:17]: Should be [1.2.3.0]

**Review by jeffhataws — CHANGES_REQUESTED** [review]

**Review by jeffhataws — APPROVED** [review]

**Review by micwade-aws — APPROVED** [review]

**Pull request merged**